ibis/backends/bigquery/tests/unit/snapshots/test_compiler/test_trailing_range_window/days/out.sql

```diff
@@ -1,3 +1,3 @@
 SELECT t0.*,
-       avg(t0.`float_col`) OVER (ORDER BY UNIX_MICROS(t0.`timestamp_col`) RANGE BETWEEN 86400000000 PRECEDING AND CURRENT ROW) AS `win_avg`
+       avg(t0.`float_col`) OVER (ORDER BY UNIX_MICROS(t0.`timestamp_col`) ASC RANGE BETWEEN 86400000000 PRECEDING AND EXTRACT(DAY from INTERVAL 0 DAY) * 86400000000 FOLLOWING) AS `win_avg`
```

Comment:
This is pretty gnarly. Is the `EXTRACT` bit actually equivalent to `CURRENT ROW` 😅 ?

Reply:
I guess so, since `0 FOLLOWING = CURRENT ROW`!